content/en/docs/reference/score-spec-reference.md

```diff
@@ -319,12 +319,10 @@ containers:
 
     livenessProbe:                          # (Optional) Liveness probe
       httpGet:                              #    - Only HTTP GET is supported
-        schema: http                        #    - Specify the schema (http or https)
```

Comment:
Should be `scheme`.
More details are here: https://github.com/score-spec/score-go/blob/3bb9a5cff9feb9f898ba5166b18eb1358516ae22/schema/schema-v1b1.json#L233C14-L233C20